tests/Endatix.Api.Tests/Endpoints/Auth/RefreshTokenTests.cs

```diff
@@ -0,0 +1,67 @@
+using FastEndpoints;
+using MediatR;
+using Endatix.Api.Endpoints.Auth;
+using Endatix.Core.Infrastructure.Result;
+using Endatix.Core.UseCases.Identity;
+using Endatix.Core.UseCases.Identity.RefreshToken;
+using Endatix.Api.Tests.TestExtensions;
+
+namespace Endatix.Api.Tests.Endpoints.Auth;
+
+public class RefreshTokenTests
+{
+    private readonly IMediator _mediator;
+    private readonly RefreshToken _endpoint;
+
+    public RefreshTokenTests()
+    {
+        _mediator = Substitute.For<IMediator>();
+        _endpoint = Factory.Create<RefreshToken>(_mediator);
+    }
+
+    [Fact]
+    public async Task HandleAsync_ValidTokens_ReturnsOkWithNewTokens()
```

Comment:
Do you think we need a test where the Bearer token is invalid and doesn't start with Bearer to see the expected response? Just a question - not a todo.

Reply:
The validator ensures the header is in the correct format. Generally we can test it but so far we don't test the validators logic and I think it's fine to go without this.